db/models/news.model.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../connection.js")
+const { response } = require("../../app.js")
+
+exports.selectTopics = () => {
+    return db.query(`SELECT * FROM topics`)
+    .then((response) => {
+        return response.rows
+    })
+    .catch((error) => {
```

Comment:
I think this catch block is unnecessary.  There are ways of making it work.  But either an error will be created by a psql error (automatic), or by a Promise rejection (custom error) that you create, and will be returned from the model, and enter the catch block you have in the controller, so arguably a catch in the model is unnecessary here.